spec/01-abstract.md

```diff
@@ -1,7 +1,19 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.
+# Problem Statement
 
-The main concept behind distributed tracing is event correlation. Event correlation is a way to correlate events from one component to the events from another. It allows to find the cause-and-effect relationship between these events. For instance – find which user action in a browser caused a failure in the business logic layer.
+Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
 
-To correlate events between components, these components need to exchange and store a piece of information called context. Typically context consists of an originating event identifier, an originating component identity and other event properties. Context has two parts. The first part is a trace context. Trace context consists of properties crucial for event correlation. The second part is correlation context. Correlation context carries user-defined properties. These properties may be helpful for correlation scenarios. But they are not required and components may choose to not carry or store them.
+Today context propagation is implemented in proprietary formats individually by each tracing tool provider. This results in siginficant interoperability problems between tracing tools than manifest in two major ways:

```

Comment:
I like what we had more as introduction. It ease into the propagation without insisting on term transaction or that it is only needed for distributed tracing tools

Reply:
Perhaps if you can incorporate that idea. We have interest to this spec outside of distributed tracing, for instance for logging or debugging.

Reply:
I want to focus on the problem first before going too deep into the specific means. We will also need visuals to explain complex mechanics. Keep in mind that not all people who read this will be tracing experts.